app/components/project-geometries/types.js

```diff
@@ -107,23 +75,33 @@ export default class TypesBase extends Component {
     geometricPropertyForType.set(target, featureCollection);
   }
 
-  @computed('geometricPropertyForType.{canonical,proposedGeometry,annotations,data}')
-  get isReadyToProceed() {
-    return isMeaningfulChange(this.get('geometricPropertyForType'));
-  }
-
   @action
   async save() {
     const model = this.get('geometricPropertyForType');
 
     try {
       await model.save();
-
       const project = await model.get('project');
-      project.save();
+      await project.save();
+
+      // bt dev and local API, currentStep does NOT seem to get recomputed here.
+      // it is determing there are no updated keys.
+      // only AFTER does it think that the keys get updated, probably after it gets fetched
+      // from the server again.
+
+      // I believe it may have something to do with how model "dirty" state is handled
+      // across implementations, since the currentSTep computed has Keys that rely on
+      // "hasDirtyAttributes"
+
+      // this is a hacky solution to get this to cooperate
+      // project.notifyPropertyChange('currentStep');
```

Comment:
this was the hacky solution? but no longer needed? since all of this is captured in the issue comments, i think the comments here in the code can/should be cleaned up.